Cross organization deployment (#309)
awslabs/aws-deployment-framework: docs/admin-guide.md

    - [Regional Bootstrapping](#regional-bootstrapping)
    - [Global Bootstrapping](#global-bootstrapping)
    - [Bootstrapping Regions](#bootstrapping-regions)

>     - [Bootstrapping accounts outside the current Organization](#Bootstrapping-accounts-outside-the-current-Organization)

Reviewer: ```suggestion
    - [Bootstrapping accounts outside the current Organization](#bootstrapping-accounts-outside-the-current-organization)
```

#### Bootstrapping accounts outside the current Organization

When you would like to deploy to an account that is outside the current Organization you should first bootstrap that account manually. You can find the CloudFormation template called `example-global-adf-cross-organization.yml` in the *adf-bootstrap/* directory. Don't forget to change the parameters before deploying this template.

Next on, we have to add permissions the external Organization permissions to use the S3 bucket and the KMS key. This can be done by adding the external Organization ID at the parameter called `ExternalOrganizationIds` in the `global.yml`  *adf-bootstrap/deployment* directory. You can add multiple Organization IDs (example: Default: orgId1, orgId2, orgId3).

> Finally you can start deploying to the external accounts by defining it with `ext-account-id://` as mentioned in the *user-guide*.

Reviewer: ```suggestion
Finally, you can start deploying to the external accounts by referencing them with `ext-account-id://${account-id}` in the deployment map as mentioned in the [user guide](./user-guide.md#targets-syntax).
```
